packages/core/src/Stores/ui-store.js

```diff
@@ -144,16 +145,20 @@ export default class UIStore extends BaseStore {
         super({ root_store, local_storage_properties, store_name });
         window.addEventListener('resize', this.handleResize);
         autorun(() => {
+            const platform = getPlatformHeader(this.root_store.common.app_routing_history);
             // TODO: [disable-dark-bot] Delete this condition when Bot is ready
-            if (isBot()) {
-                document.body.classList.remove('theme--dark');
-                document.body.classList.add('theme--light');
-            } else if (this.is_dark_mode_on) {
-                document.body.classList.remove('theme--light');
-                document.body.classList.add('theme--dark');
-            } else {
+            if (platform === 'DBot') {
                 document.body.classList.remove('theme--dark');
                 document.body.classList.add('theme--light');
+                return;
+            } else if (platform === 'DTrader') {
```

Comment:
We can remove this if statement as all other platforms (read: Trader, and MT5) have dark mode.

Reply:
ahh okay. Thanks 🙏